Assets/Scripts/SS3D/Systems/Inventory/Subsystem/Clothing/Items/ClothingAsset.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace SS3D.Core.Systems.Inventory.Subsystem.Clothing.Items
```

Comment:
These namespaces go very deep. Could we get just leave out `Systems` and `Subsystem`? I feel like they don't add to the clarity of the hierarchy.